README.md

```diff
@@ -8,26 +8,40 @@ This project contains:
 - [lsp4jakarta](/lsp4jakarta) - Language Server for Jakarta EE
 - [jakarta-eclipse](/jakarta-eclipse) - Eclipse JDT LS extension and Eclipse IDE client that consumes the Language Server for Jakarta EE
 
+## Pre-requisites
+
+1. Ensure that you have installed [JavaSE-11](https://www.oracle.com/ca-en/java/technologies/javase-jdk11-downloads.html)
+
+2. Ensure that you have [Maven](https://maven.apache.org/download.cgi) installed
+
+3. Ensure that you have installed [Eclipse](https://www.eclipse.org/downloads/)
```

Comment:
Can we combine these into one point: 
JDK 11, Maven, and Eclipse are required to build `lsp4jakarta` and `jakarta-eclipse`.

Lets also put the pre-requisite section as a subheading under "Getting Started"